### PR TITLE
Issue 1556 proposed fix

### PR DESCRIPTION
**Pull request by collin-wicker**
Updated juniper_junos_show_chassis_hardware.textfsm and associated test files as proposed fix for issue 1556. With the updates the parsed output contains all item numbers, version number, part numbers, serial numbers, and description for each item listed in the test file command output.

### CODE REVIEW

File: ntc_templates/templates/juniper_junos_show_chassis_hardware.textfsm [hunk 1]
```diff
@@ -1,35 +1,97 @@
-Value Filldown FPC (\d+)
-Value Filldown MIC (\d+)
-Value Filldown PIC (\d+)
-Value XCVR (\d+)
-Value PART (\S+)
-Value SERIAL (\S+)
-Value DESCRIPTION (\S+)
+Value CHASSIS_SN (\w+)
```
Comment (jvanderaa) [inline ntc_templates/templates/juniper_junos_show_chassis_hardware.textfsm:1]:
```suggestion
Value CHASSIS_SERIAL_NUMBER (\w+)
```

File: ntc_templates/templates/juniper_junos_show_chassis_hardware.textfsm [hunk 2]
```diff
@@ -1,35 +1,97 @@
...
+Value CHASSIS_DESC (.*)
```
Comment (jvanderaa) [inline ntc_templates/templates/juniper_junos_show_chassis_hardware.textfsm:2]:
```suggestion
Value CHASSIS_DESCRIPTION (.*)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Looking good, just looking to be a bit more explicit on a few things as we chatted some.

**Comment by collin-wicker** [comment]
I have unabbreviated the value names as requested to make the results more explicit. Please review the latest commit for these changes.

**Comment by collin-wicker** [comment]
Tests have failed because I forgot to update the names in the test files. I will update the test files and commit.

**Review by jvanderaa — APPROVED** [review]

**Pull request merged**